tests/test_multiple_inheritance.py

```diff
@@ -472,3 +472,23 @@ def test_pr3635_diamond_f():
     assert o.get_f_e() == 5
 
     assert o.get_f_f() == 6
+
+
+def test_python_inherit_from_mi():
+    class PYMVF(m.MVF):
```

Comment:
On chat Aaron told me this is redundant, pulled out for focused debugging.
Maybe add a comment to that effect? E.g.

`# This test is redundant but was useful for focused debugging while working on PR #3659`